docs/namespace_config_spec.md

```diff
@@ -0,0 +1,319 @@
+# The "Ory Permission Language" Specification
```

Comment:
Naming wise it is important to have Ory in it. Keto might be renamed to Ory Permission Service at some point and it should fit in the other terminology we have (Ory Session, Ory Identity Schema, …)

Reply:
Let's go with Ory Permission Language. Noone really writes OPL any more, it should be fine

Reply:
Good points. IBM also has an [Optimization Programming Language](https://www.ibm.com/docs/en/icos/20.1.0?topic=opl-optimization-programming-language) that still seems active. Maybe we go with "Ory Permission Format" (OPF) or "Ory Permission Grammar" (OPG)? That's also better googleable.

We chose TypeScript mostly because it's popular. In the future, we might support additional programming languages since not everybody likes, wants, or uses TypeScript (and the language has so many warts). So, maybe we encode the language used in the name of the config format?

- OP-TS or OPTS: Ory Permissions as TypeScript
- OP-GO or OPGO: Ory Permissions as Go
- OP-RS or OPRS: Ory Permissions as Rust
- OP-RB or OPRB: Ory Permissions as Ruby
- OP-PY or OPPY: Ory Permissions as Python